code/modules/client/preference/preferences.dm

```diff
@@ -1191,7 +1191,7 @@ GLOBAL_LIST_INIT(special_role_times, list( //minimum age (in days) for accounts
 			if(!tweak || !istype(gear) || !(tweak in gear.gear_tweaks))
 				return
 			var/metadata = tweak.get_metadata(user, get_tweak_metadata(gear, tweak))
-			if(!metadata || !CanUseTopic(user))
+			if(!metadata)
```

Comment:
This was a `CanUseTopic` check in the OOC preference menu, for a loadout system we dont even use (gear tweaks). Honestly, this should be axed at some point.